drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -216,129 +403,128 @@ static int channel_get(const struct device *dev, enum sensor_channel chan,
 	return 0;
 }
 
-static int sample_fetch(const struct device *dev, enum sensor_channel chan)
+static int bq274xx_sample_fetch(const struct device *dev, enum sensor_channel chan)
 {
 	struct bq274xx_data *data = dev->data;
-	int ret = 0;
+	int ret;
```

Comment:
In case an unsupported channel is used:
```suggestion
	int ret = -ENOTSUP;
```

Reply:
done